react/testing_4/src/redux/Food/actions.js

```diff
@@ -0,0 +1,27 @@
+import { getFoods } from "../../services/FoodService";
```

Comment:
Should we use r-r? I'm not sure which part of the training is this card in, It would be good not to introduce too many concepts in a single place. What do you think @mvbattan @WFolini?

Reply:
I think I'd let this as is and not use r-r, so we keep concepts a bit more tangible for doing tests